src/images/about-us/torontoSkyline.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="2692" height="473" viewBox="0 0 2692 473" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this file not part of "burger menu" so ideally should have come via different PR

Reply:
Did not work on this file -lnes = + lines. Just includes accepted changes from previous PR to fix confilcts.